### DISCUSSION TITLE
JSBSim model for Bombardier Global 5000

### DISCUSSION THREAD

**Discussion by gasilva** [JSBSim-Team/jsbsim#430] [General]
Does somebody have a JSBSim model for Bombardier Global 5000 aircraft?

There is one in the paper:
[https://www.researchgate.net/publication/303902873_Conceptual_Flight_Dynamic_Model_for_Bombardier_Global_5000_Aircraft](https://www.researchgate.net/publication/303902873_Conceptual_Flight_Dynamic_Model_for_Bombardier_Global_5000_Aircraft)

**Comment by gasilva** [comment]
I manage to implement it with aeromatic...

**Comment by seanmcleod** [comment]
How about contacting the authors of the paper, their email addresses are listed. Seems to have been a university related project so I'd imagine they would be willing to share their JSBSim model.

**Reply by gasilva** [reply]
yes. I will do that

**Reply by gasilva** [reply]
Actually I manage to implement the Global 5000 aircraft - based on the open public paper - and its engine BR710 - that is already in database - in JSBSim. Do you like to make it available in girhub? I can make a pull request.

**Comment by seanmcleod** [comment]
Sure, submit a pull request.

**Reply by gasilva** [reply]
OK. It is there.
